plugin/pki/crl_util.go

```diff
@@ -186,7 +185,7 @@ func buildCRL(ctx context.Context, b *backend, req *logical.Request, forceNew bo
 			// TODO: In this case, remove it and continue? How likely is this to
 			// happen? Alternately, could skip it entirely, or could implement a
 			// delete function so that there is a way to remove these
-			return errutil.InternalError{Err: "found revoked serial but actual certificate is empty"}
+			return errutil.InternalError{Err: fmt.Sprintf("found revoked serial but actual certificate is empty")}
```

Comment:
This  fmt.Sprintf is not necessary, there is a golangci-lint  error(S1039: unnecessary use of fmt.Sprintf (gosimple)) as it is now, please remove it.

Reply:
Agreed - I was questioning to remove that however as I was cloning from the latest PKI Plugin from Hashicorp into the monitor so I left it (please see line 195: https://github.com/hashicorp/vault/blob/main/builtin/logical/pki/crl_util.go) - i'll tidy